s31321-JakubTrzaskowski/Pamietnik_1.txt

```diff
@@ -1,9 +1,56 @@
-Drogi pamiętniczku dziś jestem w Warszawie na konferencji czyli w końcu mam
-chwilę zająć się ukosem i zrobić zadania.
-Dzięki nim nauczyłem się:
-- Tworzyć, usuwać katalogi i pliki,
-- Tworzyć maszynę wirtualną,
-- Przełączać się pomiędzy środowiskiem graficznym i tesktowym,
-- Wyświetlać zawartość katalogów,
-- Przemieszczać się pomiędzy katalogami za pomocą ścieżki względnej i
-bezwzględnej.
+30 września 2023:
+
+Dzisiaj nauczyłem/am się:
+
+Procesu zaliczenia przedmiotu.
+Poruszania się po terminalu oraz podstawowych komend.
+Korzystania z "szuflandii".
+Używania i zarządzania GitHubem oraz repozytoriami.
+Podstawowych komend w systemie Linux.
+14 października 2023:
```

Comment:
Dobrze by było dodać jakąś linię separacyjną PRZED datą kolejnego wpisu gdyż wtedy ta zlewa się z poprzednim wpisem. Wizualnie gorzej się to czyta.